examples/tomo/README.md

```diff
@@ -2,25 +2,53 @@
 
 These examples demonstrate the capability of ODL to perform tomographic projections, back-projections and filtered back-projection reconstruction in the various supported geometries. They also serve as copy-and-paste templates for the basic setup of more complex applications in tomography.
 
-## List of examples
+## Basic usage examples
+
+### 2D
+
+Example | Purpose | Complexity
+------- | ------- | ----------
+[`ray_trafo_parallel_2d.py`](ray_trafo_parallel_2d.py) | Projection and back-projection in 2D parallel beam geometry | middle
+[`ray_trafo_parallel_2d_complex.py`](ray_trafo_parallel_2d_complex.py) | Projection and back-projection in 2D parallel beam geometry **with complex-valued data** | middle
+[`skimage_ray_trafo_parallel_2d.py`](skimage_ray_trafo_parallel_2d.py) | Projection and back-projection in 2D parallel beam geometry **using the `scikit-image` back-end** | middle
```

Comment:
Shouldn't this be removed now? Given that we try to hide impl as good as possible